pkg/resources/deployment.go

```diff
@@ -0,0 +1,74 @@
+package resources
+
+import (
+	appsv1beta1 "k8s.io/api/apps/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func Deployment(namespace string) *appsv1beta1.Deployment {
+	return &appsv1beta1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "ark",
+			Namespace: namespace,
```

Comment:
We should probably label the deployment itself with component=ark too. Not sure offhand if we do that in our yaml...